pkg/coreos/templates/containerd/run-command.sh.tpl

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# initiliaze default containerd config if does not exist
+if [ ! -s /etc/containerd/config.toml ]; then
+    mkdir -p /etc/containerd/
+    /run/torcx/unpack/docker/bin/containerd config default > /etc/containerd/config.toml
+    chmod 0644 /etc/containerd/config.toml
+fi
```

Comment:
Are you saying this is flatcar specific? We use this exact mechanism to configure `containerd` on gardenlinux, right? 

So the process here is that your scripts allow to start docker (and with it `containerd` using `/etc/containerd/config.toml`), then the bootstrap successfully pulls the hyperkube image to extract the kubelet and stuff, then operatingsystemconfig is pulled down and updates `/etc/containerd/config.toml` with our custom pause image (and potentially other stuff). Is that a correct summary of what's going on?

Reply:
I guess ubuntu is doing it in the same way? https://github.com/gardener/gardener-extension-os-ubuntu/blob/master/pkg/generator/templates/cloud-init-ubuntu.template#L62

Reply:
Garden Linux was done long time ago, but now when I look at it, it completely rely on the gardener to create the file `/etc/containerd/config.toml` and the same issue might happen there. I guess previously docker was using own containerd+runc instead of relying on the those installed via the runc and containerd packages.

And yes, I faced this issue exactly with Ubuntu where node bootstrap was failing because containerd was not working because the `/etc/containerd/config.toml` was missing.